spec/index.bs

```diff
@@ -1623,6 +1623,19 @@ applications should generally request the "worst" limits that work for their con
         which are uniform buffers.
         See [=Exceeds the binding slot limits=].
 
+    <tr><td><dfn>maxFragmentCombinedOutputResources</dfn>
+        <td>{{GPUSize32}} <td>[=limit class/maximum=] <td>8
+    <tr class=row-continuation><td colspan=4>
+        The maximum total number of
+        {{GPUShaderStage/FRAGMENT}}-stage storage buffers,
+        {{GPUShaderStage/FRAGMENT}}-stage storage textures,
+        and color attachments in a {{GPURenderPipelineDescriptor}}.
```

Comment:
and depth attachments

Reply:
is there anywhere we're tracking WebGPU limits vs WebGL2 limits by GPU?

WebGL2 has a minimum limit of 4 color attachments but most desktops seem to support 8 + depth + stencil (so 9 or 10 depending on how you count)

I'd be good to know the maximum for this setting for a given GPU  matches or exceeds WebGL2

Reply:
> ~and depth attachments~

Never mind, Vulkan doesn't say depth attachments are included.

> is there anywhere we're tracking WebGPU limits vs WebGL2 limits by GPU?

Not currently, but the "correspondence" document would be an appropriate place for it. This limit doesn't have a corresponding WebGL limit because it's specific to storage buffers/textures, but maxColorAttachments has a base value of 8 in WebGPU.